src/test/java/suite/operator/V2ApiOperatorTest.java

```diff
@@ -28,6 +31,48 @@ public class V2ApiOperatorTest {
 
     private static final String SITE_ID = EnvUtil.getEnv("UID2_E2E_SITE_ID");
 
+    @ParameterizedTest(name = "{index} ==> Sender {0} encrypts with {1}, recipient {2} decrypts with {3}, expected result is {4}")
+    @MethodSource({
+            "suite.operator.TestData#sharingArgs",
```

Comment:
it's a pity the test data is far away from the test

Reply:
Yeah I initially had the test data in the same class but thought I should follow the convention.